docs/products/flink.rst

```diff
@@ -1,86 +1,42 @@
-
-
-Get started with Aiven for Apache Flink
----------------------------------------
-
-Take your first steps with Aiven for Apache Flink by following our :doc:`/docs/products/flink/getting-started` article, or browse through our full list of articles:
+Aiven for Apache Flink® |beta|
+================================
 
+Aiven for Apache Flink® is a fully managed service that leverages the power of the `open-source Apache Flink framework <https://flink.apache.org/>`_ to provide distributed, stateful stream processing capabilities, allowing users to perform real-time computation with SQL efficiently.
 
 .. grid:: 1 2 2 2
 
-    .. grid-item-card::
+    .. grid-item-card:: :doc:`Quickstart </docs/products/flink/getting-started>`
         :shadow: md
         :margin: 2 2 0 0
 
-        📚 :doc:`Concepts </docs/products/flink/concepts>`
+        Get started with Aiven for Apache Flink and integrate data services.
 
-    .. grid-item-card::
+    .. grid-item-card:: :doc:`Overview </docs/products/flink/list-overview>`
         :shadow: md
         :margin: 2 2 0 0
 
-        💻 :doc:`/docs/products/flink/howto`
+        Discover Aiven for Apache Flink features, architecture, plans, and pricing.
 
-    .. grid-item-card::
+    .. grid-item-card:: :doc:`Concepts </docs/products/flink/concepts>`
         :shadow: md
         :margin: 2 2 0 0
 
-        📖 :doc:`/docs/products/flink/reference`
-
-
-Apache Flink features
----------------------
-
-**Flink SQL**
-  Apache Flink enables you to develop streaming applications using standard SQL. The :doc:`Aiven web console provides an SQL editor </docs/products/flink/concepts/supported-syntax-sql-editor>` to explore the table schema and create SQL queries to process streaming data.
-
-**Built-in data flow integration with Aiven for Apache Kafka®**
-  Connect with Aiven for Apache Kafka® as a source or sink for your data.
-
-  * Autocompletion for finding existing topics in a connected Kafka service when you create data tables.
-  * Choose the table format when reading data from Kafka - JSON, Apache Avro, Confluent Avro, Debezium CDC.
-  * Supports :doc:`upsert Kafka connectors </docs/products/flink/concepts/kafka-connectors>`, which allow you to produce a changelog stream, where each data record represents an update or delete event.
-
-**Built-in data flow integration with Aiven for PostgreSQL®**
-  Connect with Aiven for PostgreSQL® as a source or sink for your data. The Aiven web console features autocompletion for finding existing databases in a connected PostgreSQL service when you create data tables.
-
-**Automate workflows**
-  Automate workflows for managing Flink services with :doc:`Aiven Terraform Provider </docs/tools/terraform>`. See the `Flink data source <https://registry.terraform.io/providers/aiven/aiven/latest/docs/data-sources/flink>`_ for details.
```

Comment:
Are we removing all the features part?

Reply:
All the features are available in the **Aiven for Apache Flink® features**, and **Aiven for Apache Flink® managed service features** sections. Removed them from the main landing page.